demo/src/app/app.module.ts

```diff
@@ -35,7 +35,6 @@ export function markedOptionsFactory(anchorService: AnchorService): MarkedOption
     ClipboardButtonModule,
     HttpClientModule,
     MarkdownModule.forRoot({
-      loader: HttpClient,
```

Comment:
The `loader` property should not be removed.

This has been added so that people don't forget to import the `HttpClientModule` when they want to load markdown from external source. The same was as ngx-translate is doing it (see their example [here](https://github.com/ngx-translate/core#configuration).

Reply:
it's different than ngx-translate
here is example
https://github.com/robertIsaac/angular-localization/blob/master/src/app/app.config.ts
I'm using a custom loader which you can find here https://github.com/robertIsaac/angular-localization/blob/master/src/app/core/utils/translate-browser.loader.ts
to change the path the files are in
and if someone forget it he will get a very clear message

that being said, if you wish to keep it, I will return it
and if that's the last thing to merge it you can return it and merge it directly instead of waiting for me

Reply:
> it's different than ngx-translate here is example https://github.com/robertIsaac/angular-localization/blob/master/src/app/app.config.ts I'm using a custom loader which you can find here https://github.com/robertIsaac/angular-localization/blob/master/src/app/core/utils/translate-browser.loader.ts to change the path the files are in and if someone forget it he will get a very clear message
> 
> that being said, if you wish to keep it, I will return it and if that's the last thing to merge it you can return it and merge it directly instead of waiting for me

Because I had open issues about it previously and the fact that people could just go _"oh we are importing the HttpClientModule in our app but we aren't using it anywhere"_ I prefer to keep  it so that it makes it more obvious that it is used by the ngx-markdown library... the same way I prefer to have providers inside the MarkdownModule configuration instead of adding providers in the providers module property.